packages/ethereum-contracts/contracts/agreements/CFAv1Forwarder.sol

```diff
@@ -0,0 +1,413 @@
+// SPDX-License-Identifier: AGPLv3
+pragma solidity >= 0.8.4;
+
+import {
+    ISuperfluid,
+    ISuperToken,
+    BatchOperation,
+    FlowOperatorDefinitions
+} from "../interfaces/superfluid/ISuperfluid.sol";
+
+import {
+    IConstantFlowAgreementV1
+} from "../interfaces/agreements/IConstantFlowAgreementV1.sol";
+
+import { CallUtils } from "../libs/CallUtils.sol";
+
+/**
+ * The CFAv1Forwarder contract provides an easy to use interface to
+ * ConstantFlowAgreementV1 specific functionality of Super Tokens.
+ * Instances of this contract can operate on the protocol only if configured as "trusted forwarder"
+ * by protocol governance.
+ */
+contract CFAv1Forwarder {
+    error CFAFwd_InvalidFlowrate();
```

Comment:
I think we should maintain our `SNAKE_CASE` standard for custom errors

Reply:
yes, let's do that. we can check what we can do in .solhint configuration, "custom error" probably is  still too early for it to support, but it does have some support of snake case enforcement for immutable/constant variables.

Reply:
took the liberty to address this quickly